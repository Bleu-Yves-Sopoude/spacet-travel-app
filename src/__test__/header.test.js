import { render } from '@testing-library/react';
import React from 'react';
import Header from '../components/header';
import { BrowserRouter } from 'react-router-dom';
import '@testing-library/jest-dom/extend-expect';

const links = [
  {
    id: 3,
    path: '/',
    navigationContent: 'Rockets',
  },
  {
    id: 2,
    path: '/missions',
    navigationContent: 'Missions',
  },
  {
    id: 1,
    path: '/profile',
    navigationContent: 'My Profile',
  },
];

const AppMock = () =>  (
    <BrowserRouter>
      <Header links={links} />
    </BrowserRouter>
  );

describe('test header', () => {
  it('should render all links', () => {
    render(<AppMock />);
    expect(Header).toMatchSnapshot();
  });
});
