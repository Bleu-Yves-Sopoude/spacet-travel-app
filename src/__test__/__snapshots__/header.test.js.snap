// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test header should render all links 1`] = `[Function]`;
